web_app/dist/title.css

```diff
@@ -0,0 +1,57 @@
+@import url('https://fonts.googleapis.com/css?family=Raleway');
+
+html {
+    
+}
+
+body {
+    background: #93ca68;
+    padding: 70px 0;
```

Comment:
Is this necessary if we're doing all the actual stuff in the other divs?

Reply:
you double commented this.  also removed.